From 3e4d4113564f300fd0bc7fecb409ffbcc590c985 Mon Sep 17 00:00:00 2001
From: Julian Wolf <juwolf@suse.com>
Date: Thu, 3 Apr 2014 14:55:19 +0200
Subject: [PATCH] udev-rules: include 59-scsi-sg3_utils.rules

Without the 59-scsi-sg3_utils.rules udev-rule a couple of devices are missing
in /dev/disk/by-id.

If the luks device is specified by-id in /etc/crypttab and rd.luks.uuid not
passed as a commandline parameter during boot, systemd tries to start its
crypto services which depend on those missing devices until it times out and
exits to dracut rescue shell.

Signed-off-by: Julian Wolf <juwolf@suse.com>
---
 modules.d/95udev-rules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/95udev-rules/module-setup.sh b/modules.d/95udev-rules/module-setup.sh
index 2d3671237c..fab4ffc960 100755
--- a/modules.d/95udev-rules/module-setup.sh
+++ b/modules.d/95udev-rules/module-setup.sh
@@ -32,6 +32,7 @@ install() {
         60-pcmcia.rules \
         50-udev.rules 95-late.rules \
         50-firmware.rules \
+        59-scsi-sg3_utils.rules \
         75-net-description.rules \
         80-net-name-slot.rules 80-net-setup-link.rules \
         "$moddir/59-persistent-storage.rules" \
-- 
1.9.3

